### PR TITLE
Deprecate old method for getting version details

### PR DESCRIPTION
**Pull request by hillalex**
* Updated model/spec to only return things we're currently using. 
* Removed old method
* Re-organised `OrderlyTests` and `OrderlyReviewerTests` into `ReportTests` and `VersionTests`

Also need to merge https://github.com/vimc/montagu-webmodels/pull/75/files and point webmodels back to master.

Haven't included dependencies or parameters, since we don't yet know how best to represent these in the portal.

### CODE REVIEW

File: src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt [hunk 1]
```diff
@@ -3,17 +3,20 @@ package org.vaccineimpact.reporting_api.tests.integration_tests.tests
...
+import org.junit.FixMethodOrder
 import org.junit.Test
+import org.junit.runners.MethodSorters
 import org.vaccineimpact.reporting_api.db.JooqContext
 import org.vaccineimpact.reporting_api.db.Tables
 import org.vaccineimpact.reporting_api.db.Tables.REPORT_VERSION
 import org.vaccineimpact.reporting_api.security.InternalUser
 import org.vaccineimpact.reporting_api.tests.insertReport
 
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```
Comment (hillalex) [inline src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt]: Just forcing these tests to run in order, given the limitations of the test data!

Comment (EmmaLRussell) [reply src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt]: Good idea!

File: src/app/src/main/kotlin/org/vaccineimpact/reporting_api/db/Orderly.kt [hunk 2]
```diff
@@ -102,82 +102,31 @@ class Orderly(isReviewer: Boolean = false) : OrderlyClient
...
                     published = reportVersionResult.published,
                     requester = reportVersionResult.requester,
-                    script = scriptResult?.filename,
-                    hashScript = scriptResult?.fileHash)
-
-            return result
-
+                    artefacts = aretefacts,
+                    resources = getResourceHashes(name, version).keys.toList(),
+                    dataHashes = mapOf())
```
Comment (EmmaLRussell) [inline src/app/src/main/kotlin/org/vaccineimpact/reporting_api/db/Orderly.kt]: There's a method in the DataController now for getting datahashes

File: src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt (hunk 1 above)
Comment (EmmaLRussell) [reply src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt]: Although this branch still seems to have failing tests in TC! Should they all be fixed now?

File: src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt [hunk 3]
```diff
@@ -3,17 +3,20 @@ package org.vaccineimpact.reporting_api.tests.integration_tests.tests
...
 class VersionTests : IntegrationTest()
 {
     @Test
-    fun `publishes report`()
+    fun `A publishes report`()
```
Comment (EmmaLRussell) [inline src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt]: What's the significance of A and B?

File: src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/database_tests/VersionTests.kt [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+import org.vaccineimpact.reporting_api.tests.insertArtefact
+import org.vaccineimpact.reporting_api.tests.insertFileInput
+import org.vaccineimpact.reporting_api.tests.insertReport
+import java.sql.Timestamp
+
+class VersionTests : CleanDatabaseTests()
+{
+
+    @Test
+    fun `reader can get published report version details`()
```
Comment (EmmaLRussell) [inline src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/database_tests/VersionTests.kt]: Should be testing data hashes too?

File: src/app/src/main/kotlin/org/vaccineimpact/reporting_api/db/Orderly.kt (hunk 2 above)
Comment (hillalex) [reply src/app/src/main/kotlin/org/vaccineimpact/reporting_api/db/Orderly.kt]: Sorry you're right,  I totally submitted this before it was ready.

File: src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt (hunk 3 above)
Comment (hillalex) [reply src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt]: Forcing them to run alphabetically so making sure that `A` runs before `B`. Those test  failures look like legit failures cos I got the schema wrong! Fixed now.

Comment (EmmaLRussell) [reply src/app/src/test/kotlin/org/vaccineimpact/reporting_api/tests/integration_tests/tests/VersionTests.kt]: Oh of course. Maybe a little comment to that effect would be helpful.

**Pull request merged**